### DISCUSSION TITLE
Auto update RD client

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#4704] [Feature Request]
Nearly at 1.2.0 and there is no way to auto update the client on an endpoint. 

Updating Windows is fine - just download and run the latest .exe. 

On a Mac it's more convoluted though - Stop the service, move the latest version of RD to Applications folder and then restart the service.

Has anyone any idea how to script the updates for macOS?

**Comment by ghost** [comment]
Hallo,
eine automatische Client-Aktualisierung würde ich auch begrüßen. Ich habe heute nur durch Zufall entdeckt, dass es eine neue Version gibt.
Es würde auch schon reichen, wenn der Client über eine neue Version informiert.

Hello,
I would also welcome an automatic client update. I just discovered by accident today that there is a new version.
It would also be enough if the client informed about a new version.

**Comment by BayJay** [comment]
It would be very convenient

**Comment by nemumaru** [comment]
I second this. A "check for update" button in the "About RustDesk" window would be great. Even better a push notification that, if needed, the user can put on snooze for e.g. 24hours.

**Comment by xlionjuan** [comment]
For macOS, you can use homebrew cask to do this, though it still not automatic.
